shared/js/participantStore.js

```diff
@@ -72,6 +75,15 @@ loop.store.ParticipantStore = function() {
     },
 
     /**
+     * Receives action from chrome space.
+     *
+     * @param {sharedActions.SetOwnDisplayName[]} actionData.data
+     */
+    updateOwnDisplayName(actionData) {
+      this.setOwnDisplayName(actionData.data[0]);
```

Comment:
adding a call to update the room info would trigger a change on the presence indicator for all the guests connected to the room

`this.updateRoomInfo({ userId: this._currentUserId });`